src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/render/MixinChunkOcclusionDataBuilder.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * This file is part of the Meteor Client distribution (https://github.com/MeteorDevelopment/meteor-client).
+ * Copyright (c) Meteor Development.
+ */
```

Comment:
fix copyright notice to fit the other files and write the original creator down below.

Reply:
also this fix was already included in legacy, so ehm, you could have also checked there.

Reply:
Thanks, was a stupid over sight from me 😅